CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+Feb 08, 2023
+
+## v1.0.3
+
+### Bug Fixes
+
+- [#846](https://github.com/crypto-org-chain/cronos/pull/846) Disable authz message
+
```

Comment:
do we mention 2 other [changes](https://github.com/crypto-org-chain/ethermint/compare/v0.20.1-cronos...crypto-org-chain:ethermint:6dc910ab1fd7)

Reply:
removed the other changes for now.

Reply:
yes, might fix main as well